gnupg.py

```diff
@@ -1428,7 +1435,7 @@ def list_keys(self, secret=False, keys=None, sigs=False):
             which = 'secret-keys'
         else:
             which = 'sigs' if sigs else 'keys'
-        args = ['--list-%s' % which, '--fingerprint', '--fingerprint']  # get subkey FPs, too
+        args = ['--list-%s' % which, '--with-keygrip', '--fingerprint', '--fingerprint']  # get subkey FPs, too
```

Comment:
Please make this more compatible by e.g. inserting the `--with-keygrip` only for GnuPG >= 2.1. I think the `FIELDS` and `valid_keywords` may not need to be conditional on GnuPG version - not sure.